Libplanet.RocksDBStore/RocksDBUtils.cs

```diff
@@ -9,23 +10,40 @@ internal static class RocksDBUtils
             DbOptions options,
             string dbPath,
             ColumnFamilies? columnFamilies = null,
-            bool @readonly = false)
+            RocksDbInstanceType type = RocksDbInstanceType.Primary)
         {
             if (!Directory.Exists(dbPath))
             {
                 Directory.CreateDirectory(dbPath);
             }
 
-            if (@readonly)
+            return type switch
             {
-                return columnFamilies is null
+                RocksDbInstanceType.Primary => columnFamilies is null
+                    ? RocksDb.Open(options, dbPath) : RocksDb.Open(options, dbPath, columnFamilies),
+                RocksDbInstanceType.ReadOnly => columnFamilies is null
                     ? RocksDb.OpenReadOnly(options, dbPath, false)
-                    : RocksDb.OpenReadOnly(options, dbPath, columnFamilies, false);
+                    : RocksDb.OpenReadOnly(options, dbPath, columnFamilies, false),
+                RocksDbInstanceType.Secondary => columnFamilies is null
+                    ? RocksDb.OpenAsSecondary(options, dbPath, CreateSecondaryPath(dbPath))
+                    : RocksDb.OpenAsSecondary(
+                        options,
+                        dbPath,
+                        CreateSecondaryPath(dbPath),
+                        columnFamilies),
```

Comment:
Then, should we manually remove the secondary db after every disposing of `RocksDB*Store`?

Reply:
Yeah, this is highly problematic as `Secondary` instances leave artifacts. Implementing automatic cleanup when disposing could also be problematic as it removes log files. Besides, under current implementation, such feature might require massive refactoring. 😐